transport/ipc/ipc_unix.go

```diff
@@ -111,6 +114,11 @@ type listener struct {
 // Listen implements the PipeListener Listen method.
 func (l *listener) Listen() error {
 	listener, err := net.ListenUnix("unix", l.addr)
+
+	if err != nil && isSyscallError(err, syscall.EADDRINUSE) {
```

Comment:
I think you need to also add syscall.EEXIST (or use the IsExist() function ,which is in older releases of Golang) -- because some systems will return EEXIST instead of EADDRINUSE.